test/integration/addressResolvers/index.test.ts

```diff
@@ -3,6 +3,12 @@ import { getCache, setCache } from '../../../src/addressResolvers/cache';
 import redis from '../../../src/helpers/redis';
 import randomAddresses from '../../fixtures/addresses';
 
+function purge() {
+  if (!redis) return;
+
+  return redis.flushDb();
+}
+
```

Comment:
One of the thing that is long due is to have a different setup files for integration/unit/e2e tests with jest, so we could move this to the setup files.

This will avoid to boot the redis instance on unit tests, where it is not needed